app/styles/tasks.css

```diff
@@ -1,3 +1,8 @@
+:root{
+  --slider-colour: #424CC1;
+  --slider-background: #CFD3FF;
```

Comment:
NIT: If we are repeating #A1D0FF could we think about keeping a element in root itself

Reply:
Fixed, thanks for pointing out :)